aspnetcore/security/authentication/configure-jwt-bearer-authentication.md

```diff
@@ -0,0 +1,342 @@
+
+Using bearer tokens as the authentication method is not really authentication but more authorization. Authentication is implemented when requesting access tokens for the first time in a UI application. Requesting data in an API can be described as delegated authorization. Using bearer tokens, you do not know who or what sent the access token and you do not know how the access token was acquired. The application can only say if the token is valid to use the requested API.
+
+## Token types
+
+There are many different types of tokens and many formats. You should not be creating access tokens or ID tokens yourself unless in testing scenarios. Creating self-made tokens and not following standards usually ends up with security problems and can only be used in closed systems. It is recommended to use OpenID Connect and/or OAuth to create access tokens for API access. 
+
+### Access tokens
+
+An access token is a string used by a client application to make requests to the server implementing an API. Access tokens can vary in format and different APIs may use different formats for the tokens.
+
+An access token can be encrypted. Access tokens should never be read or interpreted by a web client or UI application holding the access token, only an API is the intended audience.
+
+Access tokens are intended solely for making requests to an API. This is normally sent to the API in the **Authorization** request header as a bearer token.
+
+See [The OAuth 2.0 Authorization Framework](https://datatracker.ietf.org/doc/html/rfc6749#section-1.4)
+
+#### Application access tokens and delegeted Access tokens
+
+When requesting or creating an access token, an application or an application acting on behalf of a user can request an access token. The tokens are **application access tokens** or **delegated access tokens**. The tokens have different claims and are handled and persisted in different ways. An application is normally persisted once in the application until it expires where the delegated access token is persisted per user, either in a cookie or in a cache on a secure server.
+
+It is recommended to use delegated user access tokens whenever a user is involved. Downstream APIs can request a delegated user access token on behalf of the authenticated user. 
+
+#### Sender constrained access tokens
+
+There are two types of access tokens: **bearer tokens** and **sender-constrained** tokens. Sender-constrained tokens require the requesting client to prove possession of a private key to use the token, ensuring the token cannot be used independently. Sender-constrained tokens can be implemented in two ways:
+
+- Demonstrating Proof-of-Possession (DPoP)
+- MTLS
+
+### ID tokens
+
+ID tokens are security tokens that confirm a user’s successful authentication. The tokens allow the client to verify the user’s identity. The secure token server issues ID tokens containing claims with user information. ID tokens are always in JWT (JSON Web Token) format.
```

Comment:
I deleted secure from secure token server. Aren't JWT servers secure? Per MS style guide, less is more.

```suggestion
ID tokens are security tokens that confirm a user’s successful authentication. The tokens allow the client to verify the user’s identity. The JWT token server issues ID tokens containing claims with user information. ID tokens are always in [JWT](https://jwt.io/introduction) format.
```

Reply:
fixed